docs/lua_tests.md

```diff
@@ -0,0 +1,24 @@
+# Lua Tests
+
+## Dependencies
+
+In order to run the lua tests locally (without docker) you need to have openresty as well as the
```

Comment:
This sentence confuses me. If we only execute "make lua-test", it will use Docker.

Reply:
I can remove it, Ive been running them without docker to play around